// html-validate/dist/es/browser.js
export { A as Attribute, C as Config, a as ConfigError, b as ConfigLoader, f as DOMNode, d as DOMTokenList, g as DOMTree, D as DynamicValue, E as EventHandler, e as HtmlElement, H as HtmlValidate, l as MetaCopyableProperty, M as MetaTable, j as NestedError, N as NodeClosed, h as NodeType, u as Parser, r as Reporter, R as ResolvedConfig, m as Rule, i as SchemaValidationError, S as Severity, c as StaticConfigLoader, n as TextClassification, k as TextContent, T as TextNode, U as UserError, V as Validator, W as WrappedError, o as classifyNodeText, P as configPresets, t as definePlugin, p as keywordPatternMatcher, w as ruleExists, q as sliceLocation, s as staticResolver, v as version } from './core.js';
export { d as defineMetadata, m as metadataHelper } from './meta-helper.js';
import 'ajv';
import 'deepmerge';
import './elements.js';
import 'fs';
import 'semver';
import 'kleur';
import '@sidvind/better-ajv-errors';
import './utils/natural-join.js';
import '@babel/code-frame';
import '@html-validate/stylish';
